data/helpers.d/backend

```diff
@@ -158,10 +158,10 @@ ynh_remove_systemd_config () {
 
 	local finalsystemdconf="/etc/systemd/system/$service.service"
 	if [ -e "$finalsystemdconf" ]; then
-		sudo systemctl stop $service
-		sudo systemctl disable $service
+		ynh_systemd_action --service_name=$service_name --action=stop
+		systemctl disable $service_name
```

Comment:
Eh there is no `$service_name` ? Shouldn't it be `$service` ?

Reply:
Indeed